src/resolver/resolver.zig

```diff
@@ -1129,6 +1129,41 @@ pub const Resolver = struct {
             return .{ .not_found = {} };
         }
 
+        if (strings.hasPrefixComptime(import_path, "file:///")) {
```

Comment:
This doesn't handle space characters or anything percent-encoded 

but it is better than status quo